src/main/java/com/nvidia/spark/rapids/jni/CaseWhen.java

```diff
@@ -16,59 +16,57 @@
 
 package com.nvidia.spark.rapids.jni;
 
-import ai.rapids.cudf.*;
+import ai.rapids.cudf.ColumnVector;
+import ai.rapids.cudf.DType;
 
 
 /**
  * Exedute SQL `case when` semantic.
  * If there are multiple branches and each branch uses scalar to generator value,
  * then it's fast to use this class because it does not generate temp string columns.
- *
+ * <p>
  * E.g.:
- *   SQL is:
- *     select
- *        case
- *          when bool_1_expr then "value_1"
- *          when bool_2_expr then "value_2"
- *          when bool_3_expr then "value_3"
- *          else "value_else"
- *        end
- *      from tab
- *
+ * SQL is:
+ * select
+ * case
+ * when bool_1_expr then "value_1"
+ * when bool_2_expr then "value_2"
+ * when bool_3_expr then "value_3"
+ * else "value_else"
+ * end
+ * from tab
```

Comment:
The original way the comment was written was easier to read.  IMO style checks should not be messing with leading whitespace within a comment, as it's likely intentional.  This comment applies to other places where leading whitespace of indented blocks was removed.